cmd/subctl/root.go

```diff
@@ -84,3 +84,7 @@ func compareFiles(file1, file2 string) (bool, error) {
 
 	return bytes.Equal(first, second), nil
 }
+
+func addNamespaceFlag(cmd *cobra.Command, namespace string) {
```

Comment:
The caller needs to pass the address of `namespace`. However I don't it makes sense to generalize the option name and description, ie we lose specific useful information, eg "namespace of the service to be exported" is better than "namespace to use".  We could pass the name/description but I don't think it's worth it since this function just wraps one lint of code. Therefore, just inline `cmd.Flags().StringVarP(...)`.